src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.XboxController;
+
+public IntakeSubsystem {
+private CANSparkFlex TopMotor;
+private CANSparkFlex BottomMotor;
+	
+private double wheelsIntakeMaxSpeed = 0.6;
+private double rollerIntakeMaxSpeed = 0.6;
+}
+  
+TopMotor = new CANSparkFlex(deviceID:10, MotorType.kBrushless);
+		TopMotor.setInverted(False);
+		TopMotor.setIdleMode(IdleMode.kCoast);
+
+
+BottomMotor = new CANSparkFlex(deviceID:11, MotorType.kBrushless);
+		BottomMotor.setInverted(True);
+		BottomMotor.setIdleMode(IdleMode.kCoast);
+
+
+public class IntakeSubsystem extends SubsystemBase {
+  private Object m_drivercontroller;
+
+  public IntakeSubsystem() {}
+
+  @Override
+  public void periodic() {
+    // This method will be called once per scheduler run
+  }
+}
```

Comment:
Remove this curly bracket and place it at the end of your file, all methods need to be within the class